fiatconnect-api.md

```diff
@@ -1794,6 +1794,71 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccountInr`
+
+`CheckingAccountInr` is a Fiat Account schema that represents accounts for INR transactions.
+
+```
```

Comment:
The situation that you are mentioning is try when the person is withdrawing funds to his own account i.e. the sender and the beneficiary are the same. 
We will ned this info if the Sender and beneficiary are not the same.

Reply:
My main concern here is that this sort KYC-related information doesn't really relate to bank account information; FiatConnect doesn't natively support sending money to or from an account that you don't own. In order to do that, you would presumably have to undergo KYC for the _other_ person's account, which isn't possible. Including all of these fields also makes this schema difficult to re-use. Could we instead use a schema like this, and only allow people to cash out to accounts that they own?

```
{
    "accountName": `string`,
    "institutionName": `string`,
    "institutionCode": `string`,
    "accountNumber": `string`,
    "fiatAccountType": `FiatAccountTypeEnum.BankAccount` ,
    "providerCode": `string`
}
```
Also, how is the `providerCode` field different from the `institutionCode` field? What does the `senderType` field represent?

Also, if a user submits a valid fiat account for, say, country `A` using this schema, that account _won't_ be able to be used for a transfer in, say, `country B`, correct? In that case, you should have a separate schema for each country. In particular, if the `quote` endpoint allows a user to submit a particular Fiat Account schema, say, `FiatAccountSchemaA` to be used for a transfer, the fiat account that that user submits should be able to be re-used for _all_ other quote requests that accept `FiatAccountSchemaA`. 

With the current state of this schema, a user could cash out in country `A`, and provide this `CheckingAccount` schema for a fiat account and successfully cash out. When that user goes to country `B`, the quote endpoint will say that they can use a `CheckingAccount` schema to cash out. The client will list all their available `CheckingAccount` fiat accounts that they've already added with the provider, and the user selects the `CheckingAccount` they used from the previous transfer. When they go to cash out, the transfer will fail, since that `CheckingAccount` isn't _actually_ usable for this transfer in a different country. To avoid this, you should have a separate schema for each country.